### ISSUE TITLE
Bootstrap3 Theme : Display Some Work on the Home Page Portfolio

### ISSUE THREAD

**Issue by MikeyGMT** [e107inc/e107#2315]
This phrase is not really that great... "Display Some Work on the Home Page Portfolio"

Any idea how to put this into a LAN... Line 421 of theme.php
`<h2>Display Some Work on the Home Page Portfolio</h2>`

It's defined within 
`$LAYOUT['modern_business_home'] =  <<<TMPL`
`TMPL;`

I suggest it should use 
define("LAN_OUR_PORTFOLIO", "Our Portfolio");
Placed in English.php so it can be used in any other theme, and defined only once. 
But this code does not accept lans that I can see...

Thanks
Mike

**Comment by Jimmi08** [comment]
You need to add this:  `e107::lan('theme');`
Look at landing-zero theme.php.

**Comment by Moc** [comment]
I don't think this one is suitable for English.php, it's not generic enough. This should be defined in the theme LAN files.

**Comment by MikeyGMT** [comment]
Hmm, ok, although there is a comment in bootstrap theme somewhere suggesting use of generics... ?

**Comment by Moc** [comment]
Ah ok. It was just my personal opinion :)

**Comment by Jimmi08** [comment]
I can confirm that this string is different for each theme, so should be in theme lang file. And with new chapter menu shortcodes is not needed anymore, because portfolio usually will use chapter name instead this string.

**Comment by MikeyGMT** [comment]
I'll leave that to themers to-do, if that's OK

**Comment by Moc** [comment]
Todo:
- Add LAN to theme language file (not generic English.php) 
- Update LAN reference in template
